SIGSEGV when using -input-handler (NULL ptr deref)

I'm assuming there's unsafe code that allows this to happen and isn't core golang, but if I put an invalid entry for `-input-handler` I can cause an immediate SIGSEGV which appears to be a NULL ptr deref... I ran into this by accident when I used `-input-handler` instead of `-input-file` ...

```
$ zdns A -input-handler derp
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x68585e]

goroutine 1 [running]:
github.com/zmap/zdns.DoLookups(0x6cc000058ba0, 0x6cc000104000, 0x0, 0x0)
	/home/someuser/zdns/lookup.go:171 +0x19e
main.main()
	/home/someuser/zdns/zdns/main.go:218 +0x1300
```
With gdb:
```
$ gdb /usr/bin/zdns
...
(gdb) r A -input-handler derp
Starting program: /usr/bin/zdns A -input-handler derp
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
[New Thread 0x7435ff27e700 (LWP 22458)]
[New Thread 0x7435fea77700 (LWP 22459)]
[New Thread 0x7435fe21b700 (LWP 22460)]
[New Thread 0x7435fd973700 (LWP 22461)]
[New Thread 0x7435fc867700 (LWP 22463)]
[New Thread 0x7435fd0a6700 (LWP 22462)]

Thread 1 "zdns" received signal SIGSEGV, Segmentation fault.
0x000000000068585e in github.com/zmap/zdns.DoLookups (g=0x74c00009eb90, c=0x74c000126000, ~r2=...)
    at /home/someuser/zdns/lookup.go:171
171	/home/someuser/zdns/lookup.go: No such file or directory.
(gdb) bt
#0  0x000000000068585e in github.com/zmap/zdns.DoLookups (g=0x74c00009eb90, c=0x74c000126000, ~r2=...)
    at /home/someuser/zdns/lookup.go:171
#1  0x00000000006a43e0 in main.main () at /home/someuser/zdns/zdns/main.go:218
(gdb) x/4i $pc
=> 0x68585e <github.com/zmap/zdns.DoLookups+414>:	mov    0x20(%rdx),%rbx
   0x685862 <github.com/zmap/zdns.DoLookups+418>:	mov    0x148(%rsp),%rsi
   0x68586a <github.com/zmap/zdns.DoLookups+426>:	mov    %rsi,(%rsp)
   0x68586e <github.com/zmap/zdns.DoLookups+430>:	mov    0x290(%rsp),%rdi
(gdb) i r rdx
rdx            0x0                 0
(gdb) 
```

This obviously isn't a blocking issue because it seems to only be reproducible if you incorrectly invoke zdns. However I though you might like to know about it.

--- a/lookup.go
+++ b/lookup.go
@@ -168,8 +168,16 @@ func DoLookups(g *GlobalLookupFactory, c *GlobalConf) error {
 
 	inHandler := GetInputHandler(c.InputHandler)
 	outHandler := GetOutputHandler(c.OutputHandler)
-	inHandler.Initialize(c)
-	outHandler.Initialize(c)
+	if inHandler == nil {
+		log.Fatal("Uknown input handler \"" + c.InputHandler + "\"")
+	} else {
+		inHandler.Initialize(c)
+	}
+	if outHandler == nil {
+		log.Fatal("Uknown outnput handler \"" + c.OutputHandler + "\"")
+	} else {
+		outHandler.Initialize(c)
+	}
 
 	// Use handlers to populate the input and output/results channel
 	go inHandler.FeedChannel(inChan, &routineWG, (*g).ZonefileInput())

